JavaAssignment/src/ravi/ClientPrimeNumberSum.java

```diff
@@ -0,0 +1,15 @@
+package ravi;
+
+public class ClientPrimeNumberSum {
+	public static void main(String[] args) {
+		PrimeNumberSum object = new PrimeNumberSum();
+		int start;
```

Comment:
Here you are simply adding a line, rest of it you have to define start variable in for a loop.